main/src/library/Int16.flix

```diff
@@ -158,6 +219,27 @@ namespace Int16 {
     /// Considers the 5 rightmost bits of `position` (`position` mod 32).
     /// The bits of x have positions: 0 (rightmost bit) - 15 (leftmost bit)
     ///
-    def bit(x: Int16, position: Int32): Int32 =
+    def getBit(x: Int16, position: Int32): Int32 =
         if ((x >>> position) % 2i16 == 0i16) 0 else 1
+
+    ///
+    /// Returns `x` with the bit at position `position` set (to 1).
+    /// Considers the 5 rightmost bits of `position` (`position` mod 32).
+    /// The bits of x have positions: 0 (rightmost bit) - 15 (leftmost bit)
+    ///
+    def setBit(x: Int16, position: Int32): Int16 = x ||| (1i16 <<< position)
```

Comment:
I added `setBit`, `clearBit` and `flipBit` for all Int types.  I needed the functionality for `reverse` and I think they go well with `getBit` too.